README.md

```diff
@@ -12,12 +12,21 @@ be found in [current status](#Current-Status).
 | Image Name                                                                                                   | OS SKU                  | Type  | Build Status                                                                                                                                                                                 | Documentation                 |
 | :---------                                                                                                   | :-----                  | :---  | :-----------                                                                                                                                                                                 | :----                         |
 | [windows-2022-build](https://github.com/tianocore/containers/pkgs/container/containers%2Fwindows-2022-build) | Windows ServerCore 2022 | Build | [![Windows 2022 Images](https://github.com/tianocore/containers/actions/workflows/Windows-2022.yaml/badge.svg)](https://github.com/tianocore/containers/actions/workflows/Windows-2022.yaml) | [Doc](Windows-2022/Readme.md) |
-| [ubuntu-20-build](https://github.com/tianocore/containers/pkgs/container/containers%2Fubuntu-20-build)       | Ubuntu 20.04            | Build | [![Ubuntu 20 Images](https://github.com/tianocore/containers/actions/workflows/Ubuntu-20.yaml/badge.svg)](https://github.com/tianocore/containers/actions/workflows/Ubuntu-20.yaml)          | [Doc](Ubuntu-20/Readme.md)    |
```

Comment:
This is not related to adding he Fedora images. Please use a separate commit (or the fixup one mentioned before)